src/utils/errors.ts

```diff
@@ -90,6 +90,19 @@ export const eDownloadFailed = (message: string) => {
     })
 }
 
+export const eSaveFailed = (message: string) => {
```

Comment:
The eDownloadFailed error will do for saving files too. So let's reuse it instead of introducing a new error.